feature-libs/organization/administration/styles/_page-template.scss

```diff
@@ -1,16 +1,16 @@
 @import 'company-page-template';
 
-// The blacklist can be used to prevent template CSS
+// The blocklist can be used to prevent template CSS
 // being generated.
-$page-template-blacklist-organization: () !default;
+$page-template-blocklist-organization: () !default;
 
-// whitelisted page template selectors are processed
-// unless (some) they are blacklisted
-$page-template-whitelist-organization: CompanyPageTemplate !default;
+// allowlisted page template selectors are processed
+// unless (some) they are blocklisted
```

Comment:
[nitpick] could you change this to `// unless (some) that are blocklisted using `$page-template-blocklist``